site/content/en/docs/installation/_index.md

```diff
@@ -206,3 +207,27 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+
+## Enabling pprof endpoints
```

Comment:
Also, the installation page is probably not the best place to hold this documentation.

Can you start a new `task` https://kueue.sigs.k8s.io/docs/tasks/

Reply:
Let's take kubernetes as inspiration https://kubernetes.io/docs/tasks/

I think it's time to divide our tasks in 2 (maybe 3?) subpages:
- Batch Admin
- Batch User

Profiling could be under "Batch Admin", but I wonder if we could add another one: "Platform Engineer/Ops"?

@cortespao any ideas?

Reply:
I discussed this with some colleagues, and the conclusion is that there are 3 personas, so we can have a subpage for each:

- `Batch Application Developer` (previously known as Batch User).
- `Batch Ops` (previously known as Batch Administrator)
- `Batch Platform Developer` (here goes the pprof doc)

But we can leave this for a follow up.
For now, it suffices to put this in the `tasks` folder and list it under Batch Admin.

Reply:
I opened https://github.com/kubernetes-sigs/kueue/issues/1007